models/issue.rb

```diff
@@ -110,6 +231,16 @@ def get_all_comments
       all
    end
 
+   def due_on
+      time = @issue[:milestone][:due_on]
+      time.nil? ? nil : time.to_date
```

Comment:
If you prefer, I think `time && time.to_date` will also work here. But I've never seen anyone else use that construct, so what you have may be clearer. There's also the option of `time ? time.to_date : nil`.